hbase-oss/src/test/resources/log4j.properties

```diff
@@ -18,4 +18,7 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c{2} (%F:%M(%L)) - %m%n
 log4j.logger.org.apache.hadoop=DEBUG
 log4j.logger.org.apache.hadoop.metrics2=WARN
-log4j.logger.org.apache.hadoop.fs=WARN
\ No newline at end of file
+log4j.logger.org.apache.hadoop.fs=WARN
+log4j.org.apache.hadoop.util=WARN
+log4j.logger.org.apache.hadoop.fs.s3a=INFO
+log4j.logger.org.apache.hadoop.hbase.oss=DEBUG
```

Comment:
The DEBUG setting for log4j.logger.org.apache.hadoop encompasses this already, but its' fine.